policy/modules/roles/unconfineduser.te

```diff
@@ -74,6 +74,8 @@ allow unconfined_t self:cap_userns all_cap_userns_perms;
 
 allow unconfined_t self:bpf { map_create map_read map_write prog_load prog_run };
 
+allow unconfined_t self:perf_event manage_perf_event_perms;
```

Comment:
We should also add the same for `unconfined_domain_type` in domain.te so that e.g. unconfined services get the perms.

Reply:
Why it would be required for generic unconfined services? Or is it just an expectation?

Reply:
I'd expect `unconfined_domain_type`s to have more-or-less the same permissions as `unconfined_t`, but maybe @wrabcak will know better.

Reply:
Yes, @WOnder93  is correct. There is a macro called "`unconfined_domain()`" which calls "`unconfined_domain_noaudit()`" which calls "`domain_unconfined()`" and this macro adds the domain to  `unconfined_domain_type` . So not only unconfined_t is "unconfined" but all types which are included in "unconfined_domain_type".

Reply:
@wrabcak, we know the perf-related permissions are required for the `unconfined_t` type, so far there was no use case for other unconfined domains. So Ondrej's question should read: Is it expected that all permissions like this one should rather be allowed for all domains in `unconfined_domain_type` instead of just `unconfined_t` because there are such expectations, given the attribute name?

Reply:
@zpytela, I don't know, I believie @milosmalik  did some testing and outcome is that only unconfined_t need it ?

Reply:
There should be no problem if we allow it for` unconfined_domain_type`. Please proceed.